src/Duel.java

```diff
@@ -0,0 +1,154 @@
+public class Duel {
+    private static boolean turn = true;
+
+    public static void startDuel(Person gamer, Person enemy, Room room) {
+        while (gamer.getCurrentHealth() > 0 && enemy.getCurrentHealth() > 0) {
+            if (turn) {
+                Menu.frame();
+                Menu.gamerHealth(gamer);
+                Menu.enemyHealth(enemy);
+                Menu.gamerTurn();
+                gamerTurn(gamer, enemy, room);
+            } else {
+                Menu.frame();
+                Menu.enemyTurn();
+                enemyTurn(enemy, gamer, room);
+            }
+            if (enemy.getCurrentHealth() <= 0) {
+                Menu.enemyDeath();
+            }
+            if (gamer.getCurrentHealth() <= 0) {
+                Menu.deathMessage();
+            }
+        }
+        turn = true;
+    }
+
+    private static void gamerTurn(Person gamer, Person enemy, Room room) {
+        String type = gamer.getWeapon().getClass().getName();
+        switch (Menu.duelInteraction(gamer, enemy)) {
+            case 1 -> {
+                MeleeAttack(type, gamer.getWeapon(), enemy, room);
+                turn = false;
+            }
+            case 2 -> {
+                MeleeThrow(type, gamer.getWeapon(), enemy, room);
```

Comment:
why here is only constructor call?
if you want use some logic in class then create function inside them and call from class object